app/controllers/posts_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
   def show
     @user = User.find(params[:user_id])
-    @post = @user.posts.find_by_id(params[:id])
+    @post = @user.posts.includes(:author).find_by_id(params[:id])
     @comments = @post.comments
```

Comment:
- Good job done so far, but you have not really fixed the `N+1` problems. To fix this, please fetch all comments for a particular posts by doing this `@post=@user.posts.includes(:comments)`. Please fix this 🙏